main/src/main/assets/full_licenses.html

```diff
@@ -261,7 +261,7 @@ <h3> Original SSLeay License</h3>
 <br> *    must display the following acknowledgement:
 <br> *    "This product includes cryptographic software written by
 <br> *     Eric Young (eay@cryptsoft.com)"
-<br> *    The word 'cryptographic' can be left out if the rouines from the library
+<br> *    The word 'cryptographic' can be left out if the routines from the library
```

Comment:
These probably get dropped in the long run, in addition to being removed from this PR.